common/src/main/java/com/wynntils/models/players/hades/HadesModel.java

```diff
@@ -239,8 +239,9 @@ public void onTick(TickEvent event) {
     public void tryResendWorldData() {
         if (!isConnected()) return;
 
-        hadesConnection.sendPacket(
-                new HCPacketUpdateWorld(Models.WorldState.getCurrentWorldName(), Models.Character.getId()));
+        hadesConnection.sendPacket(new HCPacketUpdateWorld(
```

Comment:
Will this mess with Hades if not all users have the same version?

Reply:
No, this pretty much does not get used anywhere. It was just an old concept, we never got to use it.